BuoyantIO/linkerd-buoyant
# Add Helm chart

Fixes #38

Initial stab at the Helm chart for the linkerd-buoyant extension. The
required parameters in `values.yaml` are `agentID`, `agentKey`,
`agentDownloadKey` and `agentClusterName`. Please correct if those are not
right.

### How to test:
```
$ helm install linkerd-buoyant --set agentClusterName=foobar --set agentID=abc --set agentKey=xyz --set agentDownloadKey=rst chart/linkerd-buoyant

NAME: linkerd-buoyant
LAST DEPLOYED: Thu Jul  1 17:41:48 2021
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing linkerd-buoyant.

Your release is named linkerd-buoyant.

To help you manage your Buoyant Cloud agent, you can install the CLI extension by running:

  curl -sL https://buoyant.cloud/install | sh

Alternatively, you can download the CLI directly via the linkerd-buoyant releases page:

  https://github.com/BuoyantIO/linkerd-buoyant/releases

To make sure everything works as expected, run the following:

  linkerd-buoyant check

Looking for more? Visit https://buoyant.io/cloud/
```

The `README.md` was generated with `bin/helm-docs` from the linkerd2
repo.

People can use this chart as it stands in this repo using the command
above, but we should publish it in a Helm repo (linkerd's Helm repo?) to
make it visible and searchable in artifacthub.io

As mentioned in the issue referred above, it remains to hook this into
CI (either here or bcloud's repo) to automatically update and publish
this chart.
